tests/nsrt_learning/strips_learning/test_effect_search_learner.py

```diff
@@ -0,0 +1,405 @@
+"""Tests for general-to-specific STRIPS operator learning."""
+
+import pytest
+
+from predicators import utils
+from predicators.nsrt_learning.segmentation import segment_trajectory
+from predicators.nsrt_learning.strips_learning.effect_search_learner import \
+    EffectSearchSTRIPSLearner
+from predicators.structs import Action, LowLevelTrajectory, Predicate, \
+    Segment, State, Task, Type
+
+longrun = pytest.mark.skipif("not config.getoption('longrun')")
+
+
```

Comment:
how much of these tests are directly copied? we should try to use parameterized testing to remove any copying

Reply:
They're all directly copied (from `test_gen_to_spec_learner.py`). I'm not sure how to use parameterized testing to overcome this; could you maybe give an example of what you mean?

Reply:
yeah here's one example: https://github.com/Learning-and-Intelligent-Systems/predicators/blob/master/tests/approaches/test_pg3_approach.py#L30 see also https://docs.pytest.org/en/6.2.x/parametrize.html